.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Release tracker
+about: Create a tracker issue for the upcoming release. Should be used by maintainers only.
+
+---
+
+<!-- 
+The title of the issue should be "Release v0.<minor>.0".
+
+Fill in the placeholders in the checklist below:
+
+- <minor> is the release minor number 
+- <llvm> is the LLVM major number
+- <branching-date> is the date of creating the release branch
+- <release-date> is date of bpftrace release
+
+For details on the release process, see docs/release_process.md.
+For LLVM release schedule, see https://llvm.org/.
+-->
+
+### Release progress
+
```

Comment:
Not blocking but maybe good to have a link to the doc: https://github.com/bpftrace/bpftrace/blob/master/docs/release_process.md#release-procedure